bin/ghe-restore

```diff
@@ -9,21 +9,22 @@
 #/ <https://enterprise.github.com/help/articles/adding-an-ssh-key-for-shell-access>
 #/
 #/ OPTIONS:
-#/   -f | --force      Don't prompt for confirmation before restoring.
-#/   -c | --config     Restore appliance settings and license in addition to
-#/                     datastores. Settings are not restored by default to
-#/                     prevent overwriting different configuration on the
-#/                     restore host.
-#/   -v | --verbose    Enable verbose output.
-#/   -h | --help       Show this message.
-#/        --version    Display version information and exit.
-#/   -s <snapshot-id>  Restore from the snapshot with the given id. Available
-#/                     snapshots may be listed under the data directory.
-#/   <host>            The <host> is the hostname or IP of the GitHub Enterprise
-#/                     instance. The <host> may be omitted when the
-#/                     GHE_RESTORE_HOST config variable is set in backup.config.
-#/                     When a <host> argument is provided, it always overrides
-#/                     the configured restore host.
+#/   -f | --force         Don't prompt for confirmation before restoring.
+#/   -c | --config        Restore appliance settings and license in addition to
+#/                        datastores. Settings are not restored by default to
+#/                        prevent overwriting different configuration on the
+#/                        restore host.
+#/        --skip-pages    Skip backup of GitHub Pages artifacts.
```

Comment:
```suggestion
#/        --skip-pages    Skip restore of GitHub Pages artifacts.
```

Reply:
The flag here is causing the backup script to not include the pages artifacts, so the current wording seems correct?.
We're planning on adding an equivalent flag added to `ghe-restore`.

Reply:
But you've just added it to `ghe-restore` 😁